packages/core/data/oot/world_mq/bottom_of_the_well_mq.yml

```diff
@@ -23,3 +23,7 @@
     "MQ Bottom of the Well Pot Side Room 2": "can_play_zelda || (has_explosives_or_hammer && has_hover_boots)"
     "MQ Bottom of the Well Pot Side Room 3": "can_play_zelda || (has_explosives_or_hammer && has_hover_boots)"
     "MQ Bottom of the Well Pot Lobby Alcove": "has_explosives_or_hammer && can_hit_triggers_distance"
+    "MQ Bottom of the Well Grass Dead-Hand 1": "can_cut_grass && (has_ranged_weapon_child || has_ranged_weapon_adult || has_explosives_or_hammer || can_use_sticks || can_use_sword)" #Many ways to lower the water.
```

Comment:
Age swap logic hasn't been done for MQ yet, but you can compact the ranged weapons into a single macro. "has_weapon_ranged" i believe it is.

Also, can child reach this with broken giant's knife?

Reply:
Fixed the macros issue.

I am unsure about the broken giant's knife though. It is basically the Kokiri Sword so I would assume so?

Reply:
BGK is shorter than the dagger

Reply:
BGK cannot reach it.